src/NHibernate.Test/NHSpecificTest/NH3848/TestFixture.cs

```diff
@@ -0,0 +1,266 @@
+﻿using System;
+using System.Collections;
+using System.Collections.Generic;
+using System.Data;
+using System.Linq;
+using System.Linq.Expressions;
+using NHibernate.Cache;
+using NHibernate.Cfg;
+using NHibernate.Cfg.MappingSchema;
+using NHibernate.Mapping.ByCode;
+using NUnit.Framework;
+
+namespace NHibernate.Test.NHSpecificTest.NH3848
+{
+	public abstract class TestFixture : TestCaseMappingByCode
```

Comment:
Shouldn't this one be dropped in favour of the `Fixture` class added by #460?

Ok, I guess this what you were referring to with your comment:

> Tests need to be adapted from the base fixture on the master branch.